example/create_rundir.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python3
+
+import fv3config
+import yaml
+config = yaml.safe_load(open('example/config.yml', 'r'))
+fv3config.write_run_directory(config, 'rundir')
```

Comment:
Why rename 'examples' to 'example'? That closes the door to adding more examples. Also, fv3config now has a command-line options for writing run directories, though I do still like seeing this example.

Reply:
The example could be deleted, since it also exists in the `fv3config` repository. I agree with rhea that there isn't a need to change `examples` (which we use in other repos also) to `example`.

Reply:
I am fine with either! I am more inclined to 2) so you don't have to learn about another repo to get something started